src/domain/communication/communication/dto/communication.discussion.updated.args.ts

```diff
@@ -0,0 +1,11 @@
+import { ArgsType, Field } from '@nestjs/graphql';
+import { UUID } from '@domain/common/scalars';
+
+@ArgsType()
+export class CommunicationDiscussionUpdatedArgs {
+  @Field(() => UUID, {
+    description: 'The Communication to receive Updates from.',
```

Comment:
receive Updates?

Reply:
Please format this into a suggestion. I am not sure what are you asking about or suggesting.